packages/ui/src/views/unified-pipeline-studio/components/unified-popover-commit-info.tsx

```diff
@@ -19,10 +19,7 @@ export const PopoverCommitInfo = {
     return (
       <div className="flex items-center justify-between p-3">
         <div className="flex items-center gap-x-3">
-          <Avatar.Root size="8" className="inline-flex">
-            {!!avatarUrl && <Avatar.Image src={avatarUrl} />}
-            <Avatar.Fallback>{initials}</Avatar.Fallback>
-          </Avatar.Root>
+          <Avatar name={initials ?? ''} src={avatarUrl} size="lg" rounded />
```

Comment:
We can probably refactor this component to not require passing the initials.
We can probably refactor this component to remove `initials` now.

```suggestion
          <Avatar name={authorName} src={avatarUrl} size="lg" rounded />
```